app/assets/src/components/views/compare/SamplesHeatmapVis.jsx

```diff
@@ -81,6 +83,9 @@ class SamplesHeatmapVis extends React.Component {
         // only display colors to positive values
         scaleMin: 0,
         printCaption: this.generateHeatmapCaptions(),
+        // Shrink to fit the viewport width
+        // See https://developer.mozilla.org/en-US/docs/Web/API/window/innerWidth
+        maxWidth: window.innerWidth,
```

Comment:
We discuss this sometime ago for other visualization. This should be size of the div that includes the heatmap svg.